Makefile

```diff
@@ -42,6 +42,7 @@ clean:
 .PHONY: clean
 
 pristine: generate fmt
+	git diff | cat
```

Comment:
What do we need this for?(Just curious

Reply:
At work, I have an old version of go that doesn't agree with the autoformatting of the one in circleci. So I needed the actual output of the diff instead of just the list of files.

Reply:
Got it. Thanks for clarifying 👍